sites/all/modules/ereol_article/ereol_article.views_default.inc

```diff
@@ -153,8 +209,11 @@ function ereol_article_views_default_views() {
     t('sidste »'),
     t('Side'),
     t('more'),
-    t('Select any filter and click on Apply to see results'),
+    t('Vælg et filter og klik på Anvend for at se resultater'),
+    t('Avancerede indstillinger'),
```

Comment:
Danish to danish translations?

Reply:
That depends on the source language. I have no idea why `drush features-update` generates source strings in Danish, but “Side” and “sidste »" above this is also in Danish.

Reply:
Texts translated to English in https://github.com/eReolen/base/pull/345/commits/c2448f3223de646afbf214307b95cc2357f18ea1.